tests/integration/test_analytical.py

```diff
@@ -44,50 +55,63 @@ def test_run(self):
             * (1.0 + 0.5 * pressure)
         )
 
-        curr_time = time_step
+        curr_time = chem_time_step
+        last_out_time = 0
 
         idx_A = 0
         idx_B = 1
         idx_C = 2
 
         # gets analytical concentrations
+        # For this loop to approximately the model solver accurately,
+        # out_time_step should be an even multiple of chem_time_step.
         while curr_time <= sim_length:
 
             initial_A = analytical_concentrations[0][idx_A]
+            initial_B = analytical_concentrations[0][idx_B]
+            C_conc = analytical_concentrations[0][idx_C]
             A_conc = initial_A * math.exp(-(k1) * curr_time)
             B_conc = (
                 initial_A
                 * (k1 / (k2 - k1))
                 * (math.exp(-k1 * curr_time) - math.exp(-k2 * curr_time))
             )
-            C_conc = initial_A * (
+            C_conc += initial_A * (
```

Comment:
```suggestion
            C_conc = initial_A * (
```
The analytical examples can be calculated from the initial concentrations and the current time, so no need to accumulate the values.

Reply:
But! The analytical test **fails** when I make this change. I am scheduling a meeting with @mattldawson and @K20shores . . .

Reply:
The analytical approximations for **C** in test_analytical.py are only accurate for initial values **A=n, B=0.0, C=0.0**:

DEBUG:acom_music_box.conditions:Returning species_concentrations = **{'A': 0.8, 'B': 0.0, 'C': np.float64(0.0)}**
DEBUG:__main__:model 0.0   analytical 0.0
DEBUG:__main__:model 0.024277255324393295   analytical 0.024277255004167133
DEBUG:__main__:model 0.04781781782583306   analytical 0.0478178172048164
DEBUG:__main__:model 0.07064400384844993   analytical 0.07064400294519357
DEBUG:__main__:model 0.09277749230299487   analytical 0.09277749113520084
DEBUG:__main__:model 0.11423930421857616   analytical 0.11423930280313188

DEBUG:acom_music_box.conditions:Returning species_concentrations = **{'A': 0.8, 'B': 0.2, 'C': np.float64(0.0)}**
DEBUG:__main__:model 0.0   analytical 0.0
DEBUG:__main__:model 0.2242772552426475   analytical 0.024277255004167133
DEBUG:__main__:model 0.24781781774656791   analytical 0.0478178172048164
DEBUG:__main__:model 0.2706440037715903   analytical 0.07064400294519357
DEBUG:__main__:model 0.29277749222846794   analytical 0.09277749113520084
DEBUG:__main__:model 0.3142393041463108   analytical 0.11423930280313188

DEBUG:acom_music_box.conditions:Returning species_concentrations = **{'A': 0.8, 'B': 0.0, 'C': np.float64(0.3)}**
DEBUG:__main__:model 0.3   analytical 0.3
DEBUG:__main__:model 0.32427725532439344   analytical 0.024277255004167133
DEBUG:__main__:model 0.34781781782583354   analytical 0.0478178172048164
DEBUG:__main__:model 0.3706440038484505   analytical 0.07064400294519357
DEBUG:__main__:model 0.3927774923029953   analytical 0.09277749113520084
DEBUG:__main__:model 0.41423930421857635   analytical 0.11423930280313188

Reply:
Removed the accumulating code. Checked in the A=n, B=0, C=0 analytical case and wrote a python comment to that effect.